org-static-blog.el

```diff
@@ -445,40 +455,51 @@ e.g. `(('foo' 'file1.org' 'file2.org') ('bar' 'file2.org'))`"
             (push (cons tag (list post-filename)) tag-tree)))))
     tag-tree))
 
+(defun org-static-blog--preview-region ()
+  "Find the start and end of the preview in the current buffer."
```

Comment:
It looks like this function could be made simpler, by first assigning a start marker and an end marker, which are `(or org-static-blog-preview-start "<p>")` and `(or org-static-blog-preview-end "<p>")`, respectively.

After that, you could forego the big decision tree and do a static search for the area between them.

But I might be wrong on that, perhaps I am missing some nuance. That's just the impression I got from reading your implementation.